.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,19 +25,23 @@ CMDCT-
 - [ ] I have manually tested this PR in the deployed cloud environment
 
 ---
+
 ### Pre-merge checklist
 <!-- Complete the following steps before merging -->
 
 #### Review
+
 - [ ] Design: This work has been reviewed and approved by design, if necessary
 - [ ] Product: This work has been reviewed and approved by product owner, if necessary
 
 #### Security
+
 _If either of the following are true, notify the team's ISSO (Information System Security Officer)._
 
 - [ ] These changes are significant enough to require an update to the SIA.
 - [ ] These changes are significant enough to require a penetration test.
+
 ---
 
 <!-- If deploying to val or prod, click 'Preview' and select template -->
-_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
+_convert to a different template: [main → val](?expand=1&template=main-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
```

Comment:
thoughts on renaming our templates to something like
`val-deployment.md`
`production-deployment.md`
Do we think it's important to specify the branch flow in the name?